### PR TITLE
vstart.sh: use bluestore as default osd objectstore backend

### PR DESCRIPTION
**Pull request by mychoxin**
Signed-off-by: mychoxin <mychoxin@gmail.com>

### CODE REVIEW

File: src/vstart.sh [hunk 1]
```diff
@@ -150,7 +150,8 @@ usage=$usage"\t--mgr_num specify ceph mgr count\n"
 usage=$usage"\t--rgw_port specify ceph rgw http listen port\n"
 usage=$usage"\t--rgw_frontend specify the rgw frontend configuration\n"
 usage=$usage"\t--rgw_compression specify the rgw compression plugin\n"
-usage=$usage"\t-b, --bluestore use bluestore as the osd objectstore backend\n"
+usage=$usage"\t-b, --bluestore use bluestore as the osd objectstore backend(default)\n"
```
Comment (xiexingguo) [inline src/vstart.sh]: space before` (default)`

Comment (mychoxin) [inline src/vstart.sh]: done, please have a look.

**Comment by mychoxin** [comment]
@xiexingguo what do you think of it ?

**Comment by xiexingguo** [comment]
http://pulpito.ceph.com/xxg-2017-09-01_07:17:41-rados-wip-xxg-testing-2017-09-01-1357-distro-basic-smithi/
http://pulpito.ceph.com/xxg-2017-09-03_12:51:11-rados-wip-xxg-testing-2017-09-01-1357-distro-basic-smithi/

**Comment by wjwithagen** [comment]
@mychoxin @xiexingguo 
I think this is going to break FreeBSD testing, since it does not have bluestore yet.

**Comment by xiexingguo** [comment]
> @mychoxin @xiexingguo
> I think this is going to break FreeBSD testing, since it does not have bluestore yet.

Sorry, the lab has no test suites for FreeBSD for now:-)
Maybe we can fix the FreeBSD test suites by explicitly enabling the **-f** option instead?

**Comment by wjwithagen** [comment]
Easiest would be to keep old behavour voor FreeBSD and make something during initialization like
```
if [ `uname`= FreeBSD ]; then
    ....
else
    .....
fi
```
And perhaps not bother with the default word.

Also i noticed that the #! shebang has been set back to `/bin/bash`
Where it should have been `/usr/bin/env bash` . But that was not your doing 
So I'll have to look for the author for that.

**Comment by wjwithagen** [comment]
Please ignore the remark about /usr/bin/env, since that commit is post your commit.

**Comment by mychoxin** [comment]
#17492

**Pull request merged**